spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  # Required by Devise when not using database_authenticatable
+  describe '#new_session_path' do
+    subject { helper.new_session_path }
+
+    it { is_expected.to eq(new_user_session_path) }
+  end
+
+  # Required to exist by default Blacklight. Does not exist because we're not
+  # using database_authenticatable
+  describe '#edit_user_registration_path' do
```

Comment:
@rschenk @whereismyjetpack we won't need the test if override the view (see above)